### PR TITLE
Add support for plural signals

### PR DESCRIPTION
**Pull request by aleixpol**
As they were recently introduced in PackageKit.

### CODE REVIEW

**Comment by aleixpol** [comment]
@Conan-Kudo this is only supported on dnf right now, maybe someone from the Fedora teams can give it a go?

**Review by dantti — APPROVED** [review]
LGTM, tho has some minor code style differences

**Comment by ximion** [comment]
You can also test this code on Debian/Ubuntu with the APT backend now (just try any get-updates or search calls) :-)

File: src/daemon.cpp [hunk 1]
```diff
@@ -29,6 +29,68 @@
 Q_LOGGING_CATEGORY(PACKAGEKITQT_DAEMON, "packagekitqt.daemon")
 Q_LOGGING_CATEGORY(PACKAGEKITQT_OFFLINE, "packagekitqt.offline")
 
+Q_DECLARE_METATYPE(PackageKit::PkPackage);
+Q_DECLARE_METATYPE(QList<PackageKit::PkPackage>);
+Q_DECLARE_METATYPE(PackageKit::PkDetail);
+Q_DECLARE_METATYPE(QList<PackageKit::PkDetail>);
+
+const QDBusArgument &operator<<(QDBusArgument &argument, const PackageKit::PkPackage &pkg){
```
Comment (ximion) [inline src/daemon.cpp:37]: style: Opening bracket should be in its own line

Comment (aleixpol) [reply src/daemon.cpp:37]: Addressed.

Comment (ximion) [reply src/daemon.cpp:37]: Did you (force)push that change? Somehow I don't see it yet...

Comment (ximion) [reply src/daemon.cpp]: It was, thanks! :-)

**Pull request merged**